### ISSUE TITLE
React and OpenWeather API

### ISSUE THREAD

**Issue by SubtleCo** [nss-day-cohort-46/How-to-Ask-for-Help#100]
#### Have you searched the other issue tickets? If not, do that first. 
- [x] Yes
- If No, stop and search.

####  REQUIRED Add Tags to Issue Ticket and Assign to Project Board
 

- [x] Added to `c46 Help-Thread` project board
- [ ] Added the `Blocker` label if this is stoping you from moving forward

#### REQUIRED Describe your issue:
>> The OpenWeather API appears to be responding to a fetch call with the actual HTML of our own index.html.
>> The response in Postman is JSON as expected 
>> Do we need to configure a proxy?

#### REQUIRED Copy/paste any error messages that appear:
Uncaught (in promise) SyntaxError: Unexpected token < in JSON at position 0

#### REQUIRED What have you googled? **_(You must have at least 3 links related to your issue)_**
- [x] 1. https://daveceddia.com/unexpected-token-in-json-at-position-0/
- [x] 2. https://daveceddia.com/create-react-app-express-backend/#configure-the-proxy
- [x] 3. https://rapidapi.com/blog/weather-app-react/

#### REQUIRED: What have you tried? Tell us everything you have tried. 
- hard coding the API call URL
- Copy/pasting URL into postman (success)
- using .text() to see the actual response

#### REQUIRED: Copy/paste link TO THE FILE on your branch that you are having issues with
https://github.com/nss-day-cohort-46/reactive-nutshell-nss-pendant-of-intellect/blob/d10013320811de972fc802df40f325a093412681/src/components/events/weather/WeatherProvider.js#L11

#### Code Snippet

```
export const WeatherProvider = props => {
    const [weather, setWeather] = useState({})

    const getWeather = (city, stateCode) => {
        return fetch(`api.openweathermap.org/data/2.5/forecast?q=nashville,tn,us&appid=${keys.weather}`)
            .then(res => res.json())
            .then(response => console.log(response))
    }

    return (
        <WeatherContext.Provider value={{
            weather, getWeather
        }}>
            {props.children}
        </WeatherContext.Provider>
    )
}
```

**Comment by jakefroeb** [comment]
look at my help ticket i just finished your fetch needs http://

**Comment by SubtleCo** [comment]
Oh sweet jiblets. Thank you.

**Comment by SubtleCo** [comment]
fixed with #98
